src/main/java/com/minecolonies/coremod/colony/ColonyManager.java

```diff
@@ -713,7 +713,7 @@ else if (targetFile.exists())
         {
             Files.copy(file.toPath(), targetFile.toPath());
         }
-        catch (IOException e)
+        catch (final IOException e)
         {
             e.printStackTrace();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1148)